src/graphql/queries.js

```diff
@@ -917,6 +917,29 @@ export const getCollectiveSourcePaymentMethodsQuery = gql`
   }
 `;
 
+export const getMutatedCollective = gql`
```

Comment:
There's no need to write another query, the one that we need to update is the one that we already use to fetch pledges: `src/pages/collective.js` > `addCollectiveData`.

Reply:
Good point, I'll try this out and update the pull request.